### PR TITLE
Cut v0.15.2

### PR DESCRIPTION
**Pull request by mxinden**
>  * [ENHANCEMENT] [amtool] Add support for stdin to check-config (#1431) 

This is included to prevent merge conflicts with #1501.

Let me know what you think? Please do not merge yet, I want to run this against the Prometheus Operator test suit.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 0.15.2 / 2018-08-14
+
+* [ENHANCEMENT] [amtool] Add support for stdin to check-config (#1431)
+* [BUGFIX] Correctly encode query strings in notifiers (#1516)
+* [BUGFIX] Add cache control headers to the API responses to avoid IE caching (#1500)
+* [BUGFIX] Avoid listener blocking on unsubscribe (#1482)
+* [BUGFIX] Fix a bunch of unhandled errors (#1501)
+* [BUGFIX] Update PagerDuty API V2 to send full details on resolve (#1483)
+* [BUGFIX] Log PagerDuty v1 response on BadRequest (#1481)
```
Comment (stuartnelson3) [inline CHANGELOG.md]: This is probably an enhancement

**Comment by stuartnelson3** [comment]
Not sure what the deal is with the DCO, it looks like what it is expecting is wrong.

File: CHANGELOG.md (hunk 1 above)
Comment (mxinden) [reply CHANGELOG.md]: :+1:

**Comment by mxinden** [comment]
> Expected "Ben Chess <bchess@users.noreply.github.com>", but got "Benjamin Chess bchess@gmail.com>".

I guess Ben Chess removed its email address from GitHub recently, thereby the bot defaults to the GitHub internal mail address. If there are no objections I would ignore this as the commit is properly signed.

**Comment by mxinden** [comment]
Prometheus Operator tests succeeded: https://github.com/coreos/prometheus-operator/pull/1791

**Pull request merged**